### PR TITLE
Issue 66: Join room UI and "Become Distributor" contract and UI

### PR DESCRIPTION
**Pull request by ghostffcode**
Screen Shot 2021-09-09 at 9 44 01 AM

### CODE REVIEW

File: packages/react-app/src/views/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  const [tipstaCost, setTipstaCost] = useState("...");
+
+  const handleJoinRoom = ({ room }) => {
+    const slugifiedRoom = slugify(room.toLowerCase(), "_");
+
+    history.push(`/room/${slugifiedRoom}`);
+  };
+
+  const becomeDistributor = async () => {
+    const value = ethers.utils.parseEther(tipstaCost);
```
Comment (farque65) [inline packages/react-app/src/views/Home.jsx:18]: Add try catch for situations where user attempts to become a distributor before connecting their wallet

**Comment by farque65** [comment]
Very crispy

File: packages/react-app/src/views/Home.jsx (hunk 1 above)
Comment (ghostffcode) [reply packages/react-app/src/views/Home.jsx:25]: I've resolved this... please re-review

**Review by farque65 — APPROVED** [review]
Solid

**Pull request merged**